Formula/notmuch.rb

```diff
@@ -1,8 +1,9 @@
 class Notmuch < Formula
   desc "Thread-based email index, search, and tagging"
   homepage "https://notmuchmail.org/"
-  url "https://notmuchmail.org/releases/notmuch-0.30.tar.xz"
-  sha256 "5e3baa6fe11d65c67e26ae488be11b320bae04e336acc9c64621f7e3449096fa"
+  url "https://notmuchmail.org/releases/notmuch-0.31.tar.xz"
+  sha256 "571fa0e1539c86612b1f2b2c80a398e08ecfef52e27ef7e48cf8e3b84fa18394"
+  license "GPL-3.0-or-later"
```

Comment:
I suppose this formula doesn’t use the part of the tree with the Go wrapper [licensed under LGPL 2.1](https://git.notmuchmail.org/git?p=notmuch;a=blob;f=contrib/go/LICENSE;h=4362b49151d7b34ef83b3067a8f9c9f877d72a0e;hb=HEAD)?